client/my-sites/upgrades/domain-management/edit-contact-info/pending-whois-update-card.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Notice from 'components/notice';
+import support from 'lib/url/support';
+
+function PendingWhoisUpdateCard( { translate } ) {
+	return (
+		<Card className="edit-contact-info__pending-whois-update-card">
+			<Notice
+				status="is-warning"
+				showDismiss={ false }
+				isCompact={ false } >
+				<h2>{ translate( 'Domain is pending contact information update' ) }</h2>
+			</Notice>
+			<p>
+				{ translate(
+					'Your change will be reflected shortly. Please note that if you made a change to your name, ' +
+					'organization, or email address, you must confirm the change by clicking the confirmation ' +
+					'link(s) in the email(s) sent to you. If you have any questions or issues, please refer to our ' +
+					'{{supporta}}support page{{/supporta}} or {{a}}contact support{{/a}}.',
+					{
+						components: {
+							a: <a href={ support.CALYPSO_CONTACT } target="_blank" rel="noopener noreferrer" />,
```

Comment:
The contact link navigates within Calypso, so I don't think it should open in a new tab by default.